app/services/data_migrations/remove_course_has_vacancies_feature_flag.rb

```diff
@@ -0,0 +1,10 @@
+module DataMigrations
+  class RemoveCourseHasVacanciesFeatureFlag
```

Comment:
This looks good. I think we usually do the migration in a follow up PR to prevent the risk of the migration from running before the feature flag references have been removed from the code.

Could we split into two PR's please?

Reply:
Oh sorry, it was a rebasing problem. The first PR is here: https://github.com/DFE-Digital/apply-for-teacher-training/pull/9170

This is the second PR, which will just be the data migrations

Reply:
Awesome! Nice work 🚀